### DISCUSSION TITLE
我有一个问题想请教, 为什么antd-mobile的form的UI不设计成 antd 一样呢?

### DISCUSSION THREAD

**Discussion by LukerSpringtree** [ant-design/ant-design-mobile#6536] [General]
其实 antd 的h5模式下的UI 也挺好看的.
为什么不统一呢

**Comment by zombieJ** [comment]
两者目标场景不一样，移动端的可视范围会比 PC 更小一些。所以 mobile 设计上会尽可能展示多的信息量，元素之间相对的会更紧凑一些。这点在表单上的差异会更为明显，简单来说就是（输入组件）字要大、（页面总体）东西要多~

**Reply by LukerSpringtree** [reply]
UI 把antd的设计搬到了移动端，
我哭死
